ruby/hangman/hangman.rb

```diff
@@ -28,37 +27,30 @@ def update_guess_array
 	end
 
 	def did_player_win
```

Comment:
are you using this method outside of the class?  if not it could be private as well